modal/image.py

```diff
@@ -2021,6 +2027,27 @@ def build_dockerfile(version: ImageBuilderVersion) -> DockerfileSpec:
             dockerfile_function=build_dockerfile,
         )
 
+    def cmd(self, cmd: list[str]) -> "_Image":
```

Comment:
Should we just accept `*args` and convert that to a list? Seems nicer that directly exposing the Dockerfile syntax, which is a little weird.

Reply:
Not sure what's best really.

Reply:
I personally prefer the list, as in `entrypoint()` and `shell()`. It feels more dockerfiley. But perhaps args is more python? Either way, we should do the same for all.

Reply:
Yeah would have preferred a pythonic interface but good point that entrypoint and shell already expose the dockerfile style; +1 for internal consistency